target/linux/ath79/dts/qca9533_tplink_cpe210.dtsi

```diff
@@ -27,94 +16,6 @@
 			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
 		};
 
-		link1 {
-			label = "tp-link:green:link1";
-			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
-		};
-
-		link2 {
-			label = "tp-link:green:link2";
-			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
-		};
-
-		link3 {
-			label = "tp-link:green:link3";
-			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
-		};
-
-		system: link4 {
-			label = "tp-link:green:link4";
-			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
-		};
-	};
-
-	keys {
-		compatible = "gpio-keys";
-
-		reset_button {
-			label = "Reset button";
-			linux,code = <KEY_RESTART>;
-			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
-		};
-	};
-};
-
-&uart {
-	status = "okay";
-};
-
-&spi {
-	status = "okay";
-	num-cs = <1>;
-
-	flash@0 {
-		#address-cells = <1>;
-		#size-cells = <1>;
```

Comment:
Those two are removed in new parent DTSI?

Reply:
I will add it back but do not believe it is needed

Reply:
If you are _sure_ that it's not needed, you can remove it. Please add a brief comment about the remove in the commit message than. Otherwise, keep it as it will be a transparent copy/paste then.

Reply:
I have added it back for now